src/split.component.ts

```diff
@@ -201,6 +207,21 @@ export class SplitComponent implements OnChanges, OnDestroy {
             this.areas.sort((a, b) => +a.orderUser - +b.orderUser);
         }
 
+        if (this.areas.length > 1) {
+            var l = this.areas.length;
+            var c = 0;
+            var sub = this.areas[0].component.sizingEnd
```

Comment:
Weird stuff that could lead to increasing number of subscriptions (refresh is called a lot of times).
'debounceTime' of 500ms make the EventEmitter emits at wrong times I imagine?
I will try to find a way to avoid a subscribtion to child eventEmitters... For me, splitArea components should notify split component directly calling a function on it because they already have it injected.